Cork/Views/Start Page/Start Page.swift

```diff
@@ -141,8 +141,8 @@ struct StartPage: View
                                         {
                                             GroupBoxHeadlineGroup(
                                                 image: "square.and.arrow.down.on.square",
-                                                title: "You have \(appState.cachedDownloadsFolderSize.convertDirectorySizeToPresentableFormat(size: appState.cachedDownloadsFolderSize)) of cached downloads",
-                                                mainText: "These files were used for installing packages.\nThey are safe to remove."
+                                                title: "\(appState.cachedDownloadsFolderSize.convertDirectorySizeToPresentableFormat(size: appState.cachedDownloadsFolderSize)) of cached downloads",
+                                                mainText: "These files were leftover from installing packages, and are safe to remove"
```

Comment:
Saying that these are 'leftover' felt clearer and more reassuring to me, than just that they were 'used' (which might prompt the question: will they be used again?)

Reply:
I think using "leftover" is a good idea, but I don't think that this wording is nice. I would prefer "These files are leftovers from the installation process"

I also think we should maintain the newline, because the first sentence is already pretty complex to comprehend, and adding a subordinate clause would confuse users

Reply:
Building on this, perhaps "These files are leftovers from package installation. They're safe to remove."?

Reply:
Hmm, that would make sense, but for some reason, it doesn't sound right to me. It's missing another noun in my opinion. While we should use your idea as a base, we should come up with another alternative 🤔

Reply:
I'm not really sure this will address your feedback, but throwing out one last idea before I move on 😆 

"These files were leftover from a previous package installation. They're safe to remove."

Reply:
I don't like the word "leftover" in this sentence 😅 One of my original ideas was "These files are leftovers from previous package installations", but I'm not sure if people wouldn't imagine food when seeing "leftovers"

Reply:
Ah sorry, I see

Reply:
One more idea, what about "These files are leftovers from completed package installations"? This makes it clear that they are both just leftovers that you can removed, and that whather process they were involved in is finished

Reply:
I really like this!

Reply:
Great!